src/SourceBuild/content/repo-projects/dotnet.proj

```diff
@@ -24,6 +24,10 @@
     <RepositoryReference Include="symreader" />
     <RepositoryReference Include="source-build-externals" Condition="'$(DotNetBuildSourceOnly)' == 'true'" />
     <RepositoryReference Include="runtime" />
+  </ItemGroup>
+
+  <!-- Stop build at this point for runtime-only platforms like iOS -->
+  <ItemGroup Condition="'$(ShortStack)' != 'true'">
```

Comment:
I think it would be better to keep the dotnet.proj file the entry-point for full stack builds. Consider changing the `RootRepo` property to `runtime`: https://github.com/dotnet/dotnet/blob/653d8fbb189909bbd3851d10606b5eb8277f9c2f/Directory.Build.props#L147

The `RootRepo` property identifies the entry point project: https://github.com/dotnet/dotnet/blob/653d8fbb189909bbd3851d10606b5eb8277f9c2f/build.proj#L16

Reply:
@mmitche opinions?